README.md

```diff
@@ -145,4 +109,7 @@ http://127.0.0.1:26657/abci_query?path=%22/cosmos.bank.v1beta1.Query/AllBalances
 agd tx bank send keplr1 agoric15ch7da0d8nvqc8hk6dguq4ext0lvskpjcwm3patf8sygm63chmpqjlzt74 1000uist -y --chain-id agoriclocal
 ```
 
-# 
\ No newline at end of file
+# Interface
+you can run `yarn dev` inside of the `ui` folder. 
```

Comment:
def need this, just noticed nowhere does it tell the dev how to run the ui 👍🏽

Reply:
Rebased and added a change per Dan's feedback.